### ISSUE TITLE
Missing bot setup directives? 

### ISSUE THREAD

**Issue by fredczj** [zulip/zulip-redmine-plugin#7]
Hi,
I am working on a zulip/redmine setup on premise. Things are going well until I try to integrate zulip-redmine plugin.
I successfully observed (in logs of redmine) message feeding the API endpoints of my zulip server (as I don't have HTTP error after some setup and debug). 
I have set up a Bot on the Zulip whose type is 'Incoming webhook'. Unfortunately, I don't see any message coming on the #redmine stream I have opened. I don't see any activity neither in the log files, but I am not sure how to watch.

I can't find information on bot setup that much, and I was wondering if there is an existing bot somewhere to import to make the plugin works? 

Thanks,
Fred

**Comment by timabbott** [comment]
@fredczj sorry for the trouble -- can you try checking whether https://github.com/zulip/zulip-redmine-plugin/issues/6#issuecomment-441292596 helps?  

I think this plugin has bitrotted somewhat; I'll see if I can find someone to work on it, but it'd be helpful to have a report on whether the advice in that comment helps.

**Comment by fredczj** [comment]
OK, Tim, part of it worked, indeed. 
Actually, it sounds like the way the HTTP object is created in this post helped:

```
        new_uri = "https://" + Setting.plugin_redmine_zulip['zulip_server'] + "/api/v1/messages"
        uri = URI.parse(new_uri)
        http = Net::HTTP.new(uri.host, uri.port)
        http.use_ssl = true

        req = Net::HTTP::Post.new(uri.request_uri)
    [...]
```
I am also not a Ruby expert. So it is difficult for me to judge if there are some regression risks on other.
I would also have used the zulip_port() function to get the port... so, all this may need to be cleaned up. I am not really sure how. 

I hope this helps! Anyway, thank you

**Comment by timabbott** [comment]
I believe this was fixed in #8, thanks again for the report!
